pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -70,6 +71,10 @@ type DesiredStateOfWorldPopulator interface {
 	HasAddedPods() bool
 }
 
+const (
+	pvcBeingDeleted = "PVC is being deleted"
```

Comment:
Nit: Thoughts on `pvcBeingDeletedErrorMessage` as a var name?

Reply:
Renaming to pvcBeingDeletedMsg